addons/web_editor/static/src/scss/bootstrap_overridden.scss

```diff
@@ -81,3 +81,7 @@ $gray-900: map-get($grays, '900') !default;
 $black: map-get($grays, 'black') !default;
```

Comment:
done ! (except for the commit reference because the bug isn't caused by a particular commit and always has been here)

Reply:
> From 16.2 the tooltips are centered differently making this
worst because of a bigger overlap.

For the commit reference, I meant from which the tooltip became centered differently (and so from which the already existing issue became worse) :wink:  FYI, I looked for it and it is this commit [1]. (Here it is maybe not really relevant but by looking for the cause, you can sometimes find other solutions)

> Steps to reproduce:

> drop a "Text" block
select it
hover the bottom border of the "50%" Height option
This commit fixes it by disabling the pointer events of the tooltip

Nitpicking but
- remove the empty line between "Steps to reproduce" and the first step
- Add a `-` before each step
- Maybe add the result of the steps like 
    `=> The tooltip flickers infinitely, along with the blue overlay on the editor`
- Add an empty line before "This commit...." and add a `.` at the end of the sentence :wink: 

It will be more clear like that :wink: 

For the commit title (sorry I didn't see it when I reviewed :grimacing: ), remove the space between `web_editor` and the `:` :wink: 

[1]: https://github.com/odoo/odoo/commit/09b720eff1a7

Reply:
fixed